db/data/board_games.js

```diff
@@ -0,0 +1,18 @@
+const apiCalls = require("../../apiCalls.js")
+
+const board_gameData = async () => {
+  const games = ['chess', 'checkers', 'jenga', 'clue', 'scrabble', 'connect four', 'candy Land', 'exploding Kittens', 'scrabble', 'life', 'guess who', 'operation', 'Scattergories', 'apples to Apples', 'monopoly', 'battleship', 'mouse trap', 'trivial Pursuit', 'charades', 'pictionary']
```

Comment:
Here is where you can feel free to add or remove any names, should work pretty straight forward if you just follow the naming convention.

Reply:
sorry but what is the actual naming convention here, I noticed `mouse trap` doesn't capitalize the second word but `trivial Pursuit` does.

Reply:
woops! I'll fix that and push it up, good call. I guess overall naming convention is really just a space between anything that has two word, upper/lowercase doesn't seem to make much difference. Maybe just all lowercase to keep it consistent?

Reply:
Good catch, Orlando! I think consistency is key here, so whichever seems to appear most often in the database as of now, should be what we go with.